_posts/en/2018-12-28-optech-review-2018-plan-2019.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Optech review of 2018 and plans for 2019'
+permalink: /en/optech-review-2018-plan-2019
+name: 2018-12-28-optech-review-2018-plan-2019
+type: posts
+layout: post
+lang: en
+version: 1
+---
+This month, Steve, Mike, James and I met to review the progress that Optech made
+in 2018 and make plans for 2019.
+
+We started Optech earlier this year because we wanted to [help companies adopt
+scaling technologies and raise the level of conversation around scaling
+Bitcoin][announcement]. This year, we’ve:
+
+- Signed up 15 companies as members, including exchanges, custodians, wallets and merchants.
+- Held [two workshops][workshops] to discuss scaling techniques with engineers
+  from member companies. Almost all attendees thought the workshops were ‘good’
+  or ‘excellent’, with an average score of 3.7/5 for the first and 4.2/5 for the
+  second.
+- Produced 1 pilot and 26 regular
```

Comment:
If we call the end-of-year special a regular newsletter, it's 27.  Also, with the inclusion of that newsletter (still in draft with December not written yet) we're currently a bit over 36,000 words---or about 104 printed page equivalents.  Maybe for that part just say "...they'd be over 100 pages long (over 35,000 words at 350 words per page)".

Reply:
even better!